src/network/networkd-ipv4acd.c

```diff
@@ -129,6 +127,12 @@ static void on_acd(sd_ipv4acd *acd, int event, void *userdata) {
 
         link = address->link;
 
+        r = sd_ipv4acd_get_address(acd, &a);
```

Comment:
this change does not seem related to the commit? it only says `network/ipv4acd: use IN4_ADDR_TO_STRING()`

Reply:
Indeed. I mixed change to the last commit. Now it is moved.